dags/monitoring.py

```diff
@@ -0,0 +1,52 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import timedelta, datetime
+from operators.gcp_container_operator import GKEPodOperator
+from utils.gcp import gke_command
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "email": ["ascholtz@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2020, 7, 1),
```

Comment:
I'm not sure if we also want to backfill some of this data, then we can set it to something earlier. Or later

Reply:
I do think it could be useful to backfill through the full history we have for main_v4, back to 2018-10-30.

Reply:
👍  Updated the timestamp